olwebsite-app/src/style-sheets/responsive/_queries.scss

```diff
@@ -0,0 +1,103 @@
+// Small devices
```

Comment:
could we just import 

```@import 'screens';```

here instead of putting it in an index file. And import queries into main.scss? index doesn't seem necessary

Reply:
that's true. good catch

Reply:
Dont see the import yet

Reply:
My last commit does it. Importing queries makes more sense than screens as that's where the mixins are at. Plus you'll still have access to the _screens variables